lib/breakfast/type.ex

```diff
@@ -21,7 +21,19 @@ defmodule Breakfast.Type do
       determined_type
     else
       _ ->
-        raise "Failed to derive type from spec: #{Macro.to_string(spec)}"
+        raise Breakfast.TypeError, """
+
+
```

Comment:
The extra new lines make it look nicer I think

Reply:
++ I always add the new line :)